consensus/polybft/stake_manager_test.go

```diff
@@ -512,3 +484,35 @@ func (d *dummyStakeTxRelayer) SendTransactionLocal(txn *ethgo.Transaction) (*eth
 func (d *dummyStakeTxRelayer) Client() *jsonrpc.Client {
 	return nil
 }
+
+func Test_MarshalBug(t *testing.T) {
```

Comment:
I'd move this test to `rlp_encoding_test.go`. It doesn't seem it belong here.

Reply:
fixed https://github.com/0xPolygon/polygon-edge/pull/1461/commits/e525f9c99b3fd4d6415723e898ae97f06bca79da